### ISSUE TITLE
Handle Daemon-engine data file paths

### ISSUE THREAD

**Issue by illwieckz** [XQF/xqf#96]
Before, Unvanquished used an idtech3-like directory (with `main` the mod name):

```
~/.Unvanquished/
~/.Unvanquished/main
~/.Unvanquished/main/map.pk3
~/.Unvanquished/screenshots/
```

Now it uses something like that (no mod name):

```
~/.unvanquished/
~/.unvanquished/pkg
~/.unvanquished/pkg/map.pk3
~/.unvanquished/screenshots/
```

So, XQF don't know how to find some data like map pictures etc.

**Comment by illwieckz** [comment]
Also they changed some paths inside the pk3s, for example `levelshots/mapname.tga` becomes `meta/mapname/mapname.tga`.

**Comment by illwieckz** [comment]
was fixed in d079323
